fairseq/training.py

```diff
@@ -469,10 +448,10 @@ def _get_run_cmd(self):
         run_cmd += self.command_line_args
         run_cmd += ["checkpoint.save_dir=" + self.out_checkpoint_dir.get_path()]
 
-        if self.use_cache_manager:
+        if self.cache_manager:
             run_cmd += ["task.data=" + self.out_cached_audio_manifest.get_path()]
 
-        run_cmd.insert(0, self.fairseq_root.get_path() + "fairseq_cli/hydra_train.py")
+        run_cmd.insert(0, os.path.join(self.fairseq_root.get_path(), "fairseq_cli/hydra_train.py"))
```

Comment:
```suggestion
        run_cmd.insert(0, os.path.join(self.fairseq_root.get_path(), "fairseq_cli", "hydra_train.py"))
```
if you already use `os.path.join` then we can have it join all the components of the path.